openfisca_france/model/prelevements_obligatoires/impot_revenu/reductions_impot.py

```diff
@@ -2475,35 +4051,57 @@ def formula_2007_01_01(foyer_fiscal, period, parameters):
 
     def formula_2009_01_01(foyer_fiscal, period, parameters):
         '''
-        Sommes versées pour l'emploi d'un salariés à  domicile
-        2009-2013
+        Sommes versées pour l'emploi d'un salariés à domicile
+        2009-2011
         '''
         nb_pac2 = foyer_fiscal('nb_pac2', period)
         f7db = foyer_fiscal('f7db', period)
         f7df = foyer_fiscal('f7df', period)
         f7dl = foyer_fiscal('f7dl', period)
-        f7dq = foyer_fiscal('f7dq', period)
-        f7dg = foyer_fiscal('f7dg', period)
+        annee1 = foyer_fiscal('f7dq', period)
+        isinvalid = foyer_fiscal('f7dg', period)
         P = parameters(period).impot_revenu.reductions_impots.salarie_domicile
 
-        isinvalid = f7dg
-        annee1 = f7dq
         nbpacmin = nb_pac2 + f7dl
         maxBase = P.max1 * not_(annee1) + P.max1_1ereAnnee * annee1
```

Comment:
Il faut privilégier le [snake case](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names) dans le nom des variables : 
`max_base`, `max_non_invalide` ...

Reply:
Cf. PR #914